src/main/java/structures/Arrayer.java

```diff
@@ -33,10 +56,22 @@ public static int[] lastOfFourDigits(int[] input) {
      * ex: [1.2, 3.6, 7.9, 4.1] -> [1, 4, 8, 4]
      */
     public static int[] roundUp(double[] input) {
-        // TODO implement roundUp()
-        return new int[0];
+        //create array
+        int[] result = new int[input.length];
+        // create loop
+        for(int i = 0; i < input.length; i++){
+            // mod 1 to get decimal
+            if (input [i] % 1 >= 0.5){
+                result[i] = (int) input [i] + 1;
+                // else statement incase number is less than 0.5
```

Comment:
be careful here - comments should go directly *above* or next to the code they describe 👀